django_rq/queues.py

```diff
@@ -32,7 +34,9 @@ def __init__(self, *args, **kwargs):
         autocommit = kwargs.pop('autocommit', None)
         self._autocommit = get_commit_mode() if autocommit is None else autocommit
 
-        return super(DjangoRQ, self).__init__(*args, **kwargs)
+        job_class = kwargs.pop('job_class', TZAwareJob)
```

Comment:
Thanks for the PR! I don't think overriding the `job_class` argument is a good idea because that will prevent people from using their own custom job class.

I think a better solution would be to create a template filter that displays the time in user's default timezone.